controllers/utils/converter.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"github.com/manifestival/manifestival"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+)
+
+func ConvertToStructuredResource(yamlContent []byte, out runtime.Object, opts ...manifestival.Option) error {
+
+	s := runtime.NewScheme()
+	RegisterSchemes(s)
```

Comment:
not sure if I followed, why is this needed here?

Reply:
So, this is creating an empty Scheme which is needed by the decoder a few lines later.
I guess, the decoder won't work correctly if the Schemes/CRDs are not present in the Scheme.